tests/nemo_text_processing/hy/data_inverse_text_normalization/test_cases_cardinal.txt

```diff
@@ -0,0 +1,23 @@
+հարյուր~100
+տասներեք~13
```

Comment:
@ekmb here not all lines contain context like you mentioned in this example(25 birds ->twenty five birds) here the majority is like 25->twenty five